### PR TITLE
Vacation

### PR DESCRIPTION
**Pull request by hrikrishnankr**
Hi,
    I have created the vacation page and hosted it in github as well.You can find the code in vacation branch.
    link : http://hrikrishnankr.github.io/Space/app/views/layout.html.

@nishintn

### CODE REVIEW

File: app/assets/JSON/approver.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[{
+	"id" : "1",
+	"name" : "Prathapan Sethu",
+	"designation" : "CEO",
```
Comment (mad-nishin) [inline app/assets/JSON/approver.json]: Why do we need designation here? Are we reusing this anywhere else?

**Comment by hrikrishnankr** [comment]
Hi,
    I can remove it.
@nishintn

File: app/controllers/vacationController.js [hunk 2]
```diff
@@ -1,3 +1,70 @@
-spaceApp.controller('vacationController',['$scope', function($scope){
+spaceApp.controller('vacationController',['$scope','User','$http', 'notifications',function($scope,User,$http,notifications){
+	var err=0,msg="",date;
+	date = new Date().getFullYear();
+	$scope.user = User.query()
+	$scope.vacations = []
+	$scope.yearArray = [date,date-1,date-2,date-3,date-4,date-5,date-6,date-7]
+	var vacationType =["Loss of Pay","Compensatory","Maternity","Paternity"];
 	
+	$http.get('../assets/JSON/approver.json').success(function(data) {
```
Comment (mad-nishin) [inline app/controllers/vacationController.js]: Why dont we move this to a service. A service to read all JSONS.

File: app/controllers/vacationController.js [hunk 3]
```diff
@@ -1,3 +1,70 @@
...
+    	$scope.approvers = data;
+    });
+
+	$http.get('../assets/JSON/vacation.json').success(function(data) {
```
Comment (mad-nishin) [inline app/controllers/vacationController.js]: Why dont we move this to a service. A service to read all JSONS.

**Comment by hrikrishnankr** [comment]
Ok

@nishintn

**Comment by mad-nishin** [comment]
@hrikrishnankr ,

This branch has too may functionality so as too much code. I recommend to split the features to as much as you can.  Create feature branch for each. 

A progressive enhancement approach.

**Pull request closed without merging**